products/suse-manager.md

```diff
@@ -0,0 +1,85 @@
+---
+title: SUSE Manager
+category: server-app
+tags: suse
+iconSlug: suse
+permalink: /suse-manager
+versionCommand: zypper info SUSE-Manager-Server-release
+releasePolicyLink: https://www.suse.com/lifecycle
+changelogTemplate: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/__RELEASE_CYCLE__/
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: General Support
+
+# Release dates are documented in each release notes in the "Version revision history" paragraph.
+# EOL dates can be found on https://www.suse.com/lifecycle.
+releases:
+-   releaseCycle: "5.0"
+    releaseDate: 2024-07-12
+    eol: 2026-06-30
+    latest: '5.0.0'
+    latestReleaseDate: 2024-07-12
+
+-   releaseCycle: "4.3"
+    releaseDate: 2022-06-20
+    eol: 2025-06-30
+    latest: '4.3.13'
+    latestReleaseDate: 2024-06-01
+
+-   releaseCycle: "4.2"
+    releaseDate: 2021-06-21
+    eol: 2023-10-31
+    latest: '4.2.14'
+    latestReleaseDate: 2023-08-29
+
+-   releaseCycle: "4.1"
+    releaseDate: 2020-07-21
+    eol: 2022-10-31
+    latest: '4.1.15'
+    latestReleaseDate: 2022-06-20
+
+-   releaseCycle: "4.0"
+    releaseDate: 2019-06-26
+    eol: 2021-06-30
+    latest: '4.0.14'
+    latestReleaseDate: 2021-06-22
+
+-   releaseCycle: "3.2"
+    releaseDate: 2018-06-25
+    eol: 2020-09-30
+    latest: '3.2.15'
+    latestReleaseDate: 2020-08-19
+
+-   releaseCycle: "3.1"
+    releaseDate: 2017-06-27
+    eol: 2019-09-30
+    latest: '3.1.11'
+    latestReleaseDate: 2019-03-01
+
+-   releaseCycle: "3.0"
+    releaseDate: 2016-04-05
+    eol: 2018-10-31
+    latest: '3.0.12'
+    latestReleaseDate: 2018-06-01
+
+-   releaseCycle: "2.1"
+    releaseDate: 2014-06-02
+    eol: 2017-04-30
+    latest: '2.1.19'
+    latestReleaseDate: 2016-10-31
+
+-   releaseCycle: "1.7"
+    releaseDate: 2012-08-09
+    eol: 2015-06-30
```

Comment:
```suggestion
    eol: true
```

Why 2015-06-30 ? Could not find any information on https://www.suse.com/lifecycle.

Reply:
I doubled checked with an other version of the page: https://web.archive.org/web/20190808082538/https://www.suse.com/lifecycle/

So EOL should be `2015-06-30`, if we wan't to keep those historic information.